examples/app_with_observability/app_async.py

```diff
@@ -7,7 +7,11 @@
     "Give me 10 names for a baby from this country {country} with gender {gender}!!!!"
 )
 
-ag.init()
+ag.init(
+    app_id="xxxxxxxx",
+    host="https://cloud.agenta.ai",
+    api_key="xxxxx.xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
+)
 tracing = ag.llm_tracing()
```

Comment:
Do we need to have this line? Can't we have it part of ag.init() and then in the call have ag.tracing.set_span_attribute? (and is this a good idea?)

Reply:
Yup, it is. I've made it this way.